qt/aqt/addons.py

```diff
@@ -665,6 +665,10 @@ def addonConfigHelp(self, module: str) -> str:
     def addonFromModule(self, module: str) -> str:
         return module.split(".")[0]
 
+    def getLogger(self, name):
+        from logging import getLogger
+        return getLogger(f"{aqt.log.LoggerManager.TAG}{self.addonFromModule(module)}")
```

Comment:
`module` is undefined. Also getLogger() is missing type hints.

Reply:
fixed